src/main/resources/patch_003.txt

```diff
@@ -0,0 +1 @@
+ALTER TABLE cg_link ADD subtype VARCHAR(50);
```

Comment:
This is not used for now, I've added it for future idea.
Also I now starting to think that INT will be better, not so nice, but occupies less space.
I propose to leave as is for now.